beatrix/src/test/java/org/killbill/billing/beatrix/integration/TestInvoicePayment.java

```diff
@@ -1172,7 +1187,7 @@ public void testWithPendingPaymentThenFailure() throws Exception {
         assertEquals(invoice1.getPayments().size(), 1);
         assertEquals(invoice1.getPayments().get(0).getAmount().compareTo(new BigDecimal("249.95")), 0);
         assertEquals(invoice1.getPayments().get(0).getCurrency(), Currency.USD);
-        assertFalse(invoice1.getPayments().get(0).isSuccess());
+        assertFalse(invoice1.getPayments().get(0).getStatus() == InvoicePaymentStatus.SUCCESS);
```

Comment:
Maybe be explicit and assert the right state here (vs `assertFalse`)?

Reply:
Fixed in https://github.com/killbill/killbill/pull/1802/commits/22883101537984447dbfd4b5fc26b183ac3a2d4c